### ISSUE TITLE
Scripting Cannot Add Only Buildings to Scene

### ISSUE THREAD

**Issue by ejboomus** [domlysz/BlenderGIS#925]
# **Describe the bug**
Attempting to import buildings with no user input. I upload a geotiff, and use bpy.ops.importgis.osm_query to import buildings. When I use filterTags to request only buildings, it states that buildings are not in the expected set. However if I either do not use filterTags or click on Get OSM first, then the code works and can import buildings. 

# **How to Reproduce**
This line of code works only after physically clicking on GIS->Web geodata-> Get OSM. If you do not follow those steps before running the script, it says that building is not part of the set. However if you run the code with no filterTags, it imports all options with no error.
bpy.ops.importgis.osm_query(filterTags = {'building'}) only works after clicking GIS->Web geodata-> Get OSM
bpy.ops.importgis.osm_query() works all the time, but imports everything (undesired)

# **Error message**
Python: Traceback (most recent call last):
  File "\Text", line 59, in <module>
  File "C:\Program Files\Blender Foundation\Blender 4.2\4.2\scripts\modules\bpy\ops.py", line 109, in __call__
    ret = _op_call(self.idname_py(), kw)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
ValueError: Converting py args to operator properties:: 'building' not found in ()
